docs/HACKING.md

```diff
@@ -139,4 +139,26 @@ opm index add --from-index $INDEX_REPOSITORY:$INDEX_TAG
 and removed from it:
 ```shell script
 opm index rm --from-index $INDEX_REPOSITORY:$INDEX_TAG
+```
+
+## Updating Git submodules
+This project contains git submodules for four components:
+1. windows-machine-config-bootstrapper
+# TODO: Check out to latest commit on release-4.6 in https://issues.redhat.com/browse/WINC-460
+2. kubernetes has been pinned to a commit before go 1.15 became a requirement using the commands below. 
+```shell script
+$ cd kubernetes
+$ git checkout f5121a6a6a02ddfafd2bfbf5201b092dc5097ab0
+```
+3. ovn-kubernetes
+4. containernetworking-plugins
+
+To update all git submodules use:
```

Comment:
You can add a make target as well to update all submodules, considering that we will be updating it frequently.